packages/gatsby-theme-newrelic/src/components/HamburgerMenu.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@emotion/core';
+
+const menuLine = (isOpen) => css`
+  width: 100%;
+  height: 3px;
+  background-color: var(--color-brand-800);
+  margin: 5px 0;
+  border-radius: 5px;
+  transition: 0.18s;
+
+  .dark-mode & {
+    background-color: var(--color-dark-800);
+  }
+
+  ${isOpen &&
+  ` 
+  :nth-child(1) {
+    transform: rotate(-45deg) translate(-3px, 8px);
+  }
+
+  :nth-child(2) {
+    opacity: 0;
+    margin: 0
+  }
+
+  :nth-child(3) {
+    transform: rotate(45deg) translate(-3px, -8px);
+  }`}
+`;
+
+const HamburgerMenu = ({ onToggle, isOpen, className }) => (
+  <button
+    aria-expanded={isOpen}
+    aria-label="Mobile Menu"
+    type="button"
+    css={css`
+      display: block;
+      background: none;
+      border: 0;
+      cursor: pointer;
+      width: 2rem;
+      outline: none;
+      padding: 0;
+    `}
+    className={className}
+    onClick={() => onToggle()}
+  >
+    <div css={menuLine(isOpen)} />
+    <div css={menuLine(isOpen)} />
+    <div css={menuLine(isOpen)} />
+  </button>
+);
+
+HamburgerMenu.propTypes = {
+  onToggle: PropTypes.func.isRequired,
+  isOpen: PropTypes.bool,
+  className: PropTypes.string,
+};
+
+HamburgerMenu.defaultProps = {
+  isOpen: false,
+};
+
+export default HamburgerMenu;
```

Comment:
Per [this comment](https://docs.google.com/presentation/d/19Sjd-m8fEkcrkd7U1XBSZTH94omaFgzumpd-UWwB7L4/edit?disco=AAAAJvLYOOc) can we make the hamburger menu a bit smaller? It feels a bit too big.

Reply:
yes! I will make those changes

Reply:
Is 1.5rem a good width?

Reply:
Maybe 😆 . I'd say play around with it until it feels right 😄 

Also I noticed the rounding on the edges is a bit funky. Here is the dev site:
<img width="90" alt="Screen Shot 2020-07-24 at 11 33 56 AM" src="https://user-images.githubusercontent.com/565661/88424019-aaf78b80-cda1-11ea-87a9-130e8082fc8b.png">

Here is this package. I do see some rounded edges, but it would be nice to have the edges be fully round like in the dev site.
<img width="52" alt="Screen Shot 2020-07-24 at 11 34 03 AM" src="https://user-images.githubusercontent.com/565661/88424025-ab902200-cda1-11ea-95b7-e1c61231dcea.png">

Reply:
I played around with it, and the corners are rounded out now!

Reply:
<img width="171" alt="Screen Shot 2020-07-24 at 2 51 40 PM" src="https://user-images.githubusercontent.com/38332422/88425330-392d3b00-cdbd-11ea-80e3-30f2b414eee0.png">